src/vi/doctruyen3q/src/eu/kanade/tachiyomi/extension/vi/doctruyen3q/DocTruyen3Q.kt

```diff
@@ -85,4 +95,48 @@ class DocTruyen3Q : WPComics(
     }
 
     override val genresSelector = ".categories-detail ul.nav li:not(.active) a"
+	
+    private val preferences: SharedPreferences =
+        Injekt.get<Application>().getSharedPreferences("source_$id", 0x0000)
```

Comment:
```suggestion
    private val preferences: SharedPreferences = getPreferences()
```
also need to import `keiyoushi.utils.getPreferences`